magenta/music/BUILD

```diff
@@ -171,7 +171,7 @@ py_library(
 py_test(
     name = "midi_io_test",
     srcs = ["midi_io_test.py"],
-    data = ["//magenta/testdata"],
+    data = ["//magenta/testdata:testdata"],
```

Comment:
No idea. Maybe I was having issues initially, but the tests pass now, so it should not have :testdata. Thanks.